internal/upload/koji/run-koji-container.sh

```diff
@@ -81,7 +81,7 @@ koji_start() {
   ${CONTAINER_RUNTIME} run -d --name org.osbuild.koji.koji --network org.osbuild.koji \
     -v "${SHARE_DIR}:/share:z" \
     -p 80:80 \
-    -p 443:443 \
+    -p 443:4343 \
```

Comment:
This is backwards, see https://docs.docker.com/config/containers/container-networking/

```suggestion
    -p 4343:443 \
```

Reply:
Fixed!